docs/src/configuration/services/mysql/_index.md

```diff
@@ -1,80 +1,80 @@
 
-Only MariaDB is available on Dedicated and Dedicated Generation 3 environments, using Galera for replication:
+### Supported versions on Dedicated environments
 
-{{< image-versions image="mariadb" status="supported" environment="dedicated" >}}
+On Dedicated and Dedicated Generation 3 environments, only MariaDB is available with Galera for replication:
 
-{{< note >}}
+{{< image-versions image="mariadb" status="supported" environment="dedicated" >}}
 
-Downgrades of MySQL or MariaDB are not supported.
-Both will update their own data files to a new version automatically but cannot downgrade them.
-If you want to experiment with a later version without committing to it use a non-production environment.
+Dedicated environments don't support any storage engine other than InnoDB.
+Tables created on Dedicated environments using the MyISAM storage engine don't replicate between cluster nodes.
+See how to [convert the engine for tables](#storage-engine).
 
-Dedicated environments do not support any storage engine other than InnoDB.
-Tables created using the MyISAM storage engine on dedicated environments will not replicate between cluster nodes.
+### Switching type and version
 
-{{< /note >}}
+If you change the service type, your data is removed.
+To switch  service types, first [export your data](#exporting-data).
+Then remove the old service and create a new one.
+Then [import your data](#importing-data) to the new service.
 
-### Deprecated versions
+You can't downgrade either MySQL or MariaDB.
+Both update their data files to a new version automatically but can't downgrade them.
+To experiment with a later version without committing to it, use a non-production environment.
 
-The following versions are available but are not receiving security updates from upstream, so their use is not recommended.
-They will be removed at some point in the future.
+{{% deprecated-versions %}}
 
 | **`mariadb`** | **`mysql`** | **`oracle-mysql`** |
 |----------------------------------|---------------|-------------------------|
 |  {{< image-versions image="mariadb" status="deprecated" >}} | {{< image-versions image="mariadb" status="deprecated" >}} | {{< image-versions image="oracle-mysql" status="deprecated" >}} |
 
-## Relationship
+## Usage example
 
-The format exposed in the ``$PLATFORM_RELATIONSHIPS`` [environment variable](/development/variables.md#platformsh-provided-variables):
+Configure your service with at least 256 MB in disk space.
 
-{{< relationship "mysql" >}}
+{{% endpoint-description type="mariadb" sectionLink="#multiple-databases" multipleText="databases" %}}
 
-## Usage example
+### MariaDB example configuration
 
-For MariaDB your `.platform/services.yaml` use `mariadb` service type:
+Service definition:
 
 {{< readFile file="src/registry/images/examples/full/mariadb.services.yaml" highlight="yaml" >}}
```

Comment:
I was going to say treat it like the app config, but I'll go back with this one. I think we should discuss it more. 

I'd add it as a tab here and in the **Usage example** section, but Hugo's limitations are frustrating to me. So it can be a comment there. I'd put it as a comment here too but the system we have for generating the files is rather complex and also frustrating. :disappointed: